qark/qarkMain.py

```diff
@@ -949,26 +957,26 @@ def main():
 
 
     try:
+        if valid_manifest_file(common.args.manifest):
+            if ((prov_exp_list is not None) or (act_exp_list is not None) or (actalias_exp_list is not None) or (serv_exp_list is not None) or (rec_exp_list is not None)):
+                common.logger.info("Until we perfect this, for manually testing, run the following command to see all the options and their meanings: adb shell am. Make sure to update qark frequently to get all the enhancements! You'll also find some good examples here: http://xgouchet.fr/android/index.php?article42/launch-intents-using-adb")
+                try:
 
-        if ((prov_exp_list is not None) or (act_exp_list is not None) or (actalias_exp_list is not None) or (serv_exp_list is not None) or (rec_exp_list is not None)):
-            common.logger.info("Until we perfect this, for manually testing, run the following command to see all the options and their meanings: adb shell am. Make sure to update qark frequently to get all the enhancements! You'll also find some good examples here: http://xgouchet.fr/android/index.php?article42/launch-intents-using-adb")
-            try:
-
-                show_exports(prov_exp_list,'provider')
-                show_exports(act_exp_list,'activity')
-                show_exports(actalias_exp_list,'alias')
-                show_exports(serv_exp_list,'service')
-                show_exports(rec_exp_list,'receiver')
+                    show_exports(prov_exp_list,'provider')
+                    show_exports(act_exp_list,'activity')
+                    show_exports(actalias_exp_list,'alias')
+                    show_exports(serv_exp_list,'service')
+                    show_exports(rec_exp_list,'receiver')
 
-                print "\nTo view any sticky broadcasts on the device:"
-                print "adb shell dumpsys activity| grep sticky\n"
+                    print "\nTo view any sticky broadcasts on the device:"
```

Comment:
You can achieve the same with a multi-line string constant
print """
To view any sticky broadcasts on the device:
adb shell dumpsys activity| grep sticky
"""